src/util.c

```diff
@@ -125,15 +122,13 @@ int process_test_results(struct ntttcp_test_endpoint *tep)
 	tepr->cpu_ps_softirq_usage = (tepr->final_cpu_ps->softirq_time - tepr->init_cpu_ps->softirq_time) / cpu_ps_total_diff;
 
 	/* calculate for counters for xml log (compatible with Windows ntttcp.exe) */
-	tepr->throughput_Bps  = (double)total_bytes / test_duration;
-	tepr->total_bytes_MB  = (double)total_bytes / DECIMAL_BASED_UNIT_M;
```

Comment:
I prefer the old style to keep them aligned

Reply:
done